packages/client/src/pages/home/home.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link, Outlet } from 'react-router-dom'
+
```

Comment:
[Предложение]
Страницы (не в учебном проекте) могут часто меняться, поэтому их должно быть удобно добавлять/удалять/изменять.
Для этого можно сделать массив объектов, где будет храниться путь и заголовок для ссылок:
```suggestion
const links = [
  { path: '/', title: 'Главная страница' },
  { path: '/login', title: 'Авторизация' },
  { path: '/signup', title: 'Регистрация' },
  { path: '/profile', title: 'Профиль' },
  { path: '/leaderboard', title: 'Таблица лидеров' },
  { path: '/forum', title: 'Форум' },
  { path: '/game', title: 'Игра' },
]
```

Reply:
сделал вместо массива объект, чтобы можно было по ключам обращаться